sdk/mysql/Azure.ResourceManager.MySql/src/MySqlFlexibleServers/Custom/RestOperations/MySqlFlexibleServerNextLinkOperationImplementation.cs

```diff
@@ -0,0 +1,683 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager.MySql.MySqlFlexibleServers;
+
+namespace Azure.Core
+{
+    internal class MySqlFlexibleServerNextLinkOperationImplementation : IOperation
```

Comment:
add TODO of adding policy applying in Azure.Core.NextLinkOperationImplementation

Reply:
Added